1.Exploratory_Data_Analysis/scripts/3.correlation_visualization.r

```diff
@@ -103,3 +103,4 @@ p <- ggplot(df, aes(x = !!X, y = !!Y)) +
     ggtitle(title)
 
 ggsave(file_path, p)
+p
```

Comment:
Consider making this variable name more descriptive with more characters (a single character oftentimes doesn't provide much context).